### PR TITLE
Us002: view job role info fe

### PR DESCRIPTION
**Pull request by JakubW-k**

### CODE REVIEW

**Review by Alicja-Kainos — APPROVED** [review]

File: src/services/JobRoleService.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { AxiosResponse } from 'axios';
+import axios, { AxiosResponse } from 'axios';
```
Comment (amordzon) [inline src/services/JobRoleService.ts]: axios isn't used

File: views/base.html [hunk 2]
```diff
@@ -16,6 +17,9 @@
     <main class="container">
         {% block content %}
         {% endblock %}
+
+        {% block details %}
```
Comment (amordzon) [inline views/base.html]: why new block details?

File: views/job-role-details.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{% extends "base.html" %} 
+{% block title %} Job role view {% endblock %} 
+{% block details %}
```
Comment (amordzon) [inline views/job-role-details.html]: you can use block content

**Review by amordzon — APPROVED** [review]
nice!

**Pull request merged**